### PR TITLE
Add OpenZeppelin implementation link in ERC777

### PR DESCRIPTION
**Pull request by frangio**
Hey there! We've implemented ERC777 in OpenZeppelin and would like to include the link in the document.

### CODE REVIEW

**Comment by frangio** [comment]
@0xjac Can you review this please?

**Comment by frangio** [comment]
Pinging @jbaylina too. A review would be appreciated so this can be automerged. :slightly_smiling_face:

**Comment by 0xjac** [comment]
@frangio Unfortunately ERC777 is already final and we are not able to make changes to the EIP such as adding content like a new reference implementation. That being said we are planning on releasing a website for ERC777 and we would love to list your implementation there.

**Comment by axic** [comment]
To be honest this PR is outstanding from before 777 became final. I also think that changing that `Implementation` section of a final ERC does not violate the final status as it is not changing the specification, but extending the list of compatible implementations.

**Comment by frangio** [comment]
I agree with @axic here. This should not qualify as changing the EIP.

**Comment by axic** [comment]
Ping @0xjac

File: EIPS/eip-777.md [hunk 1]
```diff
@@ -1233,11 +1233,13 @@ then only the `Sent`, `Minted` and `Burned` events MUST be considered.
 
 The [repository with the reference implementation][0xjac/ERC777] contains all the [tests][ref tests].
 
-## Implementation
+## Implementations
```
Comment (axic) [inline EIPS/eip-777.md]: Actually EIP-1 prescribes "Implementation".

**Comment by axic** [comment]
Ping @0xjac @jbaylina

**Comment by axic** [comment]
@Souptacular @MicahZoltu @lightclient any editorial opinion on this?

I'm not even sure the implementation section is all that useful, as it is bound to be outdated, especially after something is final.

**Comment by MicahZoltu** [comment]
I dislike the implementations section of EIPs unless it is an inline implementation in some language (e.g., Python reference implementation of a hashing algorithm or a reference implementation of a Solidity contract).

I also agree that this is a non-normative change so OK to include in a final EIP.

I also am wary about allowing people to add implementations like this because it could end up being used as an advertising platform.

I also think OZ implementations are usually very high quality and people *should* use them so we *should* be advertising them to users.

So all in all.... 🤷

**Comment by lightclient** [comment]
Also not a huge fan of the implementation section. I think it can generally be discovered outside the EIP and removes the chronic need for these PRs to update with new impls.

**Comment by MicahZoltu** [comment]
Oh, I just realized one of the authors already declined to merge this PR so I'm going to close it.  Ultimately, it is up to the author in these situations since we don't have a strict policy.

**Pull request closed without merging**